### DISCUSSION TITLE
pymeasure module not found issue

### DISCUSSION THREAD

**Discussion by alexpowell1996** [pymeasure/pymeasure#527] [Q&A]
Hi, I am getting the following error when running my PyMeasure code.
![pymeasure issue](https://user-images.githubusercontent.com/94606668/142391131-9fc74d23-59b1-42b2-9dbd-2de82b0c6a6d.PNG)

I have ensured that pymeasure is install correctly via a pip command. I have installed PyVisa in the same way and it works fine.  Any guidance would be much appreciated.

Cheers

Alex

**Comment by bilderbuchi** [comment]
Please try renaming your `../Python/pymeasure.py` file - I think this module hides the pymeasure package's existence on your current pythonpath.

**Comment by bilderbuchi** [comment]
Also, I'm moving this to discussions as it fits better over there.
